codegenerator/cli/src/cli_args/interactive_init/mod.rs

```diff
@@ -118,8 +118,9 @@ pub async fn prompt_missing_init_args(
         None => {
             // TODO: input validation for name
             Text::new("Name your indexer:")
-                .with_default("My Envio Indexer")
+                .with_default("my_envio_indexer")
```

Comment:
```suggestion
                .with_default("my-envio-indexer")
```

Reply:
Unfortunately it fails postgres validation hence my question about that 👍🏼

Reply:
I removed the validation in https://github.com/enviodev/hyperindex/pull/184